package.json

```diff
@@ -78,7 +79,6 @@
     }
   },
   "dependencies": {
-    "@bugfender/common": "^1.1.2",
-    "@bugfender/sdk": "^2.2.1"
+    "@bugfender/common": "^1.1.2"
```

Comment:
Should common also be part of the peerDependencies?

Reply:
I've moved common to peer deps.